imap_processing/tests/ialirt/unit/test_process_hit.py

```diff
@@ -114,20 +127,26 @@ def test_prefixes():
     assert HIT_PREFIX_TO_RATE_TYPE["SLOW_RATE"] == expected_slow_rate
 
 
-def test_find_groups(xarray_data):
-    """Tests find_groups"""
-
-    filtered_data = find_groups(xarray_data)
+def test_create_l1(xarray_data):
+    """Tests create_l1"""
```

Comment:
Super nit: can you add `()` at the end of the function name to make this slightly easier to read?